sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -414,6 +399,22 @@ class Analyzer(
     }
 
     /**
+     * Build a project list for Project/Aggregate and expand the star if possible
+     */
+    private def buildExpandedProjectList(
+        exprs: Seq[NamedExpression],
+        plan: LogicalPlan): Seq[NamedExpression] = {
```

Comment:
minor: name it `child` seems better, as the `expandStarExpression` also name the `LogicalPlan` parameter `child`.